librz/util/str.c

```diff
@@ -1211,6 +1211,9 @@ RZ_API int rz_str_unescape(char *buf) {
 			buf[i] = (ch << 4) + ch2;
 			esc_seq_len = 4;
 			break;
+		case '\0':
+			buf[i] = '\0';
+			return i;
```

Comment:
Use `RZ_LOG_WARN` to add a warning here

Reply:
mmm why?

Reply:
because there's no reason for the single slash to be there at the end (unless it's a line continuation char thus the warning) ... the warning is low-prio ofc

Reply:
because there's no reason for the single slash to be there at the end (unless it's a line continuation char thus warning and not error) ... the warning is low-prio ofc